code/gateway/dataaccess.q

```diff
@@ -153,7 +165,31 @@ returntab:{[input;tab;reqno]
 
 // Generates a dictionary of `tablename!mindate;maxdate
 partdict:{[input]
-    tabname:input[`tablename];
+    tabname:input`tablename;
+
+    // Check if any serverid by servertype is striped    
+    if[any anystriped:any each stripedbyservertype:exec{all`skeysym`skeytime in key x}each attributes by servertype from .gw.servers;
+        allstriped:all each stripedbyservertype;
+        allstripedservertypes:select from .gw.servers where servertype in where allstriped;
+        anystripedandnotallstriped:select from .gw.servers where(not{all`skeysym`skeytime in key x}each attributes)&
+            servertype in where anystriped&not allstriped;
+        unstripedservertypes:select from .gw.servers where servertype in where not anystriped;
+        servers:allstripedservertypes,select from(anyandunstriped:anystripedandnotallstriped,unstripedservertypes)where i=(first;i)fby servertype;
+        // Create a dictionary of the attributes against serverids
+        // asc sorts it by serverid
+        procdict:((exec serverid from allstripedservertypes),
+            asc value exec serverid by servertype from anyandunstriped)!(exec attributes from servers)@\:`date;
+        // If the response is a dictionary index into the tablename
```

Comment:
I actually have no clue what this is doing. I only copied the line of code from the one below so that it matches with the one below. This code was already in the master branch. I have reverted the changes to the master branch and re added my changes in commit 70646eef89878dd5ef41d0654e77cec1b4ac9826. Hope it shows that the code below was not edited.